src/prefect/runner/submit.py

```diff
@@ -94,30 +97,56 @@ async def _submit_flow_to_runner(
         )
         response.raise_for_status()
 
-        flow_run_id = response.json()["flow_run_id"]
+        flow_run_id = uuid.UUID(response.json()["flow_run_id"])
 
-        return uuid.UUID(flow_run_id)
+        return await client.read_flow_run(flow_run_id)
```

Comment:
Just curious about the thinking behind returning a FlowRun object instead of a UUID, is that purely for DX?

Reply:
i was thinking about results / deps

like it'd be nice to be able to say
```python

flow_runs = submit_to_runner(...)

wait_for_submitted_runs()

results = [run.state.result() for run in flow_runs]
```

Reply:
I haven't thought too much beyond what would happen in a user flow after calling our utils but it is nice that they can pull results more easily. I think that's a good direction to go in.